### ISSUE TITLE
Link purpose in context - home page and record page

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3083]
**Accessible display**
In the home and record page the purpose of the link to a new page should be descriptive and related to the page it links. 

**Current display**
In the home and record page the link is not descriptive.

**Impact**
 Blind screen reader users cannot read the surrounding context or the link. If the link is not descriptive then they don't know where the link will redirect them.

**Options to fix**
- [ ]   replace 'Read more' in Statement on language in description as: 'Read more about statement on language in description'. 
Or keep 'Read more' and add an 'aria-label'='about statement on language in description'
- [ ] replace 'Read more' in PUL open dataset as 'Read more about PUL Open Dataset' . 
Or keep 'Read more' and add an 'aria-label'='about PUL Open Dataset'.
- [ ] Update the 'Read more' link in the record page. Same fix as the 'Statement on language in description' in the home page.

**Manual Testing**
Please make sure to test with a screen reader.

**Additional info**
For more information and options on how to update the link purpose in context see https://www.w3.org/TR/WCAG21/#link-purpose-in-context
